src/main/java/wolox/training/repositories/UserRepository.java

```diff
@@ -16,4 +18,15 @@
      */
     Optional<User> findByUsername(String username);
 
+    /**
+     * Method that obtain all users if your birthdate be between two date
+     *
+     * @param maxDate:        Max date ({@link LocalDate})
+     * @param minDate:        Min date ({@link LocalDate})
+     * @param nameContaining: Name containing of the user (String)
+     * @return {@link User} list
+     */
+    List<User> findByBirthdateLessThanEqualAndBirthdateGreaterThanEqualAndNameContainingIgnoreCase(
```

Comment:
You can do this a little bit shorter with between:

https://docs.spring.io/spring-data/data-jpa/docs/current/reference/html/#jpa.query-methods.query-creation

But this is ok.